src/ClimaAtmos/Parameterizations/SurfaceFluxes/src/SurfaceFluxes.jl

```diff
@@ -0,0 +1,324 @@
+"""
+    SurfaceFluxes
+
+  Surface flux functions, e.g., for buoyancy flux,
+  friction velocity, and exchange coefficients.
+
+## Sub-modules
+  - module Byun1990
+  - module Nishizawa2018
+
+## Interface
+  - [`compute_buoyancy_flux`](@ref) computes the buoyancy flux
+  - In addition, each sub-module has the following functions:
+    - [`compute_MO_len`](@ref) computes the Monin-Obukhov length
+    - [`compute_friction_velocity`](@ref) computes the friction velocity
+    - [`compute_exchange_coefficients`](@ref) computes the exchange coefficients
```

Comment:
I like the singleton dispatch idea, and I'm certainly open to changing this but, I'd like to record that one advantage I see from using sub-packages is that it will disallow accidentally mixing the use of functions from two different references.